tests/test.bats

```diff
@@ -258,3 +258,33 @@ _healthcheck_wait ()
 	fin rm -f
 	rm -f .docksal/docksal-local.env
 }
+
+@test "Check Pantheon Integration" {
+	[[ $SKIP == 1 ]] && skip
+
+	### Setup ###
+	docker rm -vf "$NAME" >/dev/null 2>&1 || true
+	docker run --name "$NAME" -d \
+		-v /home/docker \
+		-v $(pwd)/../tests:/var/www \
+		-e SECRET_TERMINUS_TOKEN \
+		"$IMAGE"
+	_healthcheck_wait
+
+	### Tests ###
+
+	# Confirm output is not empty and token is passed to container
+	run docker exec -it -u docker "$NAME" bash -c 'source $HOME/.docksalrc >/dev/null 2>&1; echo "${SECRET_TERMINUS_TOKEN}"'
+	[[ "${output}" != "" ]]
```

Comment:
I would do `[[ "${output}" != "${SECRET_TERMINUS_TOKEN}" ]]` here. For that to work correctly, you may need to drop the `-t` or both `-it` options.